pkg/config/engine/containerd/config_v1.go

```diff
@@ -22,13 +22,29 @@ import (
 	"github.com/pelletier/go-toml"
 
 	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/engine"
+	cfgtoml "github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
 // ConfigV1 represents a version 1 containerd config
 type ConfigV1 Config
 
 var _ engine.Interface = (*ConfigV1)(nil)
 
+type ctrdCfgV1Runtime struct {
+	tree *cfgtoml.Tree
+}
+
+var _ engine.RuntimeConfig = (*ctrdCfgV1Runtime)(nil)
+
+func (c *ctrdCfgV1Runtime) GetBinPath() string {
+	if c.tree != nil {
+		if binPath, ok := c.tree.GetPath([]string{"options", "BinaryName"}).(string); ok {
+			return binPath
+		}
+	}
+	return ""
+}
```

Comment:
```suggestion
func (c *ctrdRuntimeConfig) GetBinPath() string {
	if c.tree == nil {
		return ""
	}	
	if binPath, ok := c.tree.GetPath([]string{"options", "BinaryName"}).(string); ok {
		return binPath
	}
	
	return ""
}
```

Reply:
Sure, thanks

Reply:
I've opted for this instead. This avoids redundant `return ""` statements

```
func (c *containerdCfgRuntime) GetBinPath() string {
	if c != nil && c.tree != nil {
		if binPath, ok := c.tree.GetPath([]string{"options", "BinaryName"}).(string); ok {
			return binPath
		}
	}
	return ""
}
```